source/beamline/PGM/PGMMonoGratingSelectionControl.h

```diff
@@ -10,10 +10,17 @@ class PGMMonoGratingSelectionControl : public AMEnumeratedControl
 	Q_OBJECT
 
 public:
+//	enum Grating {
+//		Low = 3,
+//		Medium = 2,
+//		High = 1,
+//		None = 0
+//	};
```

Comment:
we might not want to keep this commented codes.

If it is possible, I still like this commented definition than the newer version.